suite/tests/api/ir_aarch64.expect

```diff
@@ -41,6 +41,27 @@ ldar   (%x1)[8byte] -> %x0
 ldarb  (%x1)[1byte] -> %w0
 ldarh  (%x1)[2byte] -> %w0
 test_ldar complete
+ldur   (%x0)[1byte] -> %b0
+ldur   +0xff(%x1)[1byte] -> %b1
+ldur   (%x2)[2byte] -> %h2
+ldur   -0x0100(%x3)[2byte] -> %h3
+ldur   (%x4)[4byte] -> %s4
+ldur   -0x0100(%x5)[4byte] -> %s5
+ldur   (%x6)[8byte] -> %d6
+ldur   -0x0100(%x7)[8byte] -> %d7
+ldur   (%x8)[16byte] -> %q8
+ldur   -0x0100(%x9)[16byte] -> %q9
+stur   %b10 -> (%x10)[1byte]
+stur   %b11 -> +0xff(%x11)[1byte]
+stur   %h12 -> (%x12)[2byte]
+stur   %h13 -> +0xff(%x13)[2byte]
+stur   %s14 -> (%x14)[4byte]
+stur   %s15 -> +0xff(%x15)[4byte]
+stur   %d16 -> (%x16)[8byte]
+stur   %d17 -> +0xff(%x17)[8byte]
+stur   %q18 -> (%x18)[16byte]
+stur   %q19 -> +0xff(%x19)[16byte]
+test_ldur_stur complete
```

Comment:
Is there a separate issue on adding tests vs a third-party AArch64 decoder, like we have for x86?  That may help to find more IR bugs.  Although if the only way to compare is the standard disassembly output, that wouldn't catch this bug, since memory sizes are not printed in normal output formats.

Reply:
There's no separate issue. It's not clear to me what the intention for such an issue would be. Is it to let others exercise the IR by writing their own use-cases/tests?

Reply:
No, the idea is to add ground-truth tests of DR's decoder/encoder/IR to DR's test suite.  We have internal-consistency tests (that the decoder, encoder, fast decoder/size decoder (for x86), INSTR_CREATE, etc. all agree with each other) for AArch64, but nothing checking that those components aren't all wrong.  For x86 we have a bunch of tests comparing the decoder to other 3rd-party decoders: comparing to binutils and to udis86.

Reply:
Ah right! Raised #4396.

Reply:
Thanks!